client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -181,9 +181,9 @@ class JetpackChecklist extends PureComponent< Props & LocalizeProps > {
 					{ isPaidPlan && productInstallStatus && (
 						<Task
 							id="jetpack_akismet"
-							title={ translate( "We're automatically turning on spam filtering." ) }
+							title={ translate( "We're automatically turning on Anti-spam." ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.